### PR TITLE
Add link to Microsoft Form to sign up to be in the educator community

### PR DESCRIPTION
**Pull request by tanhakabir**
The button uses the same styling as:
image

Would be awesome to get your feedback on the form as well! https://forms.office.com/r/22c2hL60uW

### CODE REVIEW

**Comment by sana-ajani** [comment]
Do you want to put this in the Educator Resources section to? Maybe we can include it also in the "Webinars" page? Like, to learn more about teaching with VS Code, join us: **insert form**

Re: the form. Love it! Few nits: 

In the description:

Whether you are new to using VS Code or _looking for ways to integrate it in your class_, this community is here to help answer your questions and to set you up for success. *(less about specifically learning VS Code and more about learning how it can be used for them)*

Please note this community is not for reporting issues or bugs with VS Code; if you do have a bug or issue, please submit an issue on our GitHub repository: *totally get this, but maybe we can say something about encouraging feature requests and brainstorming how the editor can be integrated with existing tools they use. Also, would put the two notes together in one paragraph*

Question 10: What most interests you about joining this community? This will help us ensure we are providing the best content for you.

Maybe reword: What would you like to gain from this community? How can we provide an engaging experience or helpful content for you?

**Comment by tanhakabir** [comment]
> Do you want to put this in the Educator Resources section to? Maybe we can include it also in the "Webinars" page? Like, to learn more about teaching with VS Code, join us: insert form

Hmm do you think it'd be discoverable there? I'm concerned that the webinars page doesn't get that much traffic

**Comment by tanhakabir** [comment]
> but maybe we can say something about encouraging feature requests and brainstorming how the editor can be integrated with existing tools they use

@sana-ajani Hmm like they should do feature requests in the community or GitHub?

**Comment by tanhakabir** [comment]
Going to close this PR for now since I'm not sure about the status of the educator community. @sana-ajani feel free to open it again if we need!
